ccleste-1.4.0.recipe

```diff
@@ -0,0 +1,53 @@
+SUMMARY="Celeste Classic C source port for 3DS and PC"
+DESCRIPTION="Celeste Classic is the original free version of Celeste, created in 2016. It was written for the Pico-8 virtual console. The game can be found on Pico-8's website and on itch.io. The original code for the game was written in a form of the Lua programming language tailored to the Pico-8, although the version of Celeste Classic found inside the full Celeste game is written in C#."
```

Comment:
This should be wrapped into multiple lines using continuations (`\`) at the end, so that each line isn't longer than 100 characters.